subworkflows/local/short_variant_calling/main.nf

```diff
@@ -1,11 +1,12 @@
-include { DEEPVARIANT                             } from '../../../modules/nf-core/deepvariant'
-include { GLNEXUS                                 } from '../../../modules/nf-core/glnexus'
-include { BCFTOOLS_CONCAT as BCFTOOLS_CONCAT_GVCF } from '../../../modules/nf-core/bcftools/concat/main'
-include { BCFTOOLS_CONCAT as BCFTOOLS_CONCAT_VCF  } from '../../../modules/nf-core/bcftools/concat/main'
-include { BCFTOOLS_FILLTAGS                       } from '../../../modules/local/bcftools/filltags/main'
-include { BCFTOOLS_NORM                           } from '../../../modules/nf-core/bcftools/norm/main'
-include { BCFTOOLS_SORT as BCFTOOLS_SORT_GVCF     } from '../../../modules/nf-core/bcftools/sort/main'
-include { BCFTOOLS_SORT as BCFTOOLS_SORT_VCF      } from '../../../modules/nf-core/bcftools/sort/main'
+//
+// Workflow to call and merge SNVs
+//
+include { BCFTOOLS_CONCAT                             } from '../../../modules/nf-core/bcftools/concat/main'
+include { BCFTOOLS_FILLTAGS                           } from '../../../modules/local/bcftools/filltags/main'
+include { BCFTOOLS_NORM as BCFTOOLS_NORM_MULTISAMPLE  } from '../../../modules/nf-core/bcftools/norm/main'
+include { BCFTOOLS_NORM as BCFTOOLS_NORM_SINGLESAMPLE } from '../../../modules/nf-core/bcftools/norm/main'
+include { DEEPVARIANT                                 } from '../../../modules/nf-core/deepvariant'
+include { GLNEXUS                                     } from '../../../modules/nf-core/glnexus'
```

Comment:
Just for simetry, it would be good to add `/main`

Reply:
Absolutely!